sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_generated/models/_enums.py

```diff
@@ -96,8 +96,6 @@ class RoleScope(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     """Global scope"""
     KEYS = "/keys"
     """Keys scope"""
-    GLOBAL_ENUM = "/"
-    """Global scope"""
```

Comment:
Was this a bug that got corrected in codegen?

Reply:
It seems to be, yeah. There definitely shouldn't be this enum value, and what's odd is that I don't find `GLOBAL_ENUM` in the swagger for any of the supported API versions. This generation is using DPG for the first time, but I wouldn't think that the old generation would have just hallucinated this value 🤔

Reply:
When there is a name conflict between something in the swagger vs a word that has significance in Python, I know we sometimes add a suffix to the generated type. I suspect that's what's happening here, but I guess I don't know why we stopped doing that for DPG. @iscai-msft ?

Reply:
Resolving this to get the PR through; I followed up with Izzy offline